docs/static/releasenotes.asciidoc

```diff
@@ -36,6 +37,34 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-8-5]]
+=== Logstash 6.8.5 Release Notes
+
+No relevant changes in Logstash core.
```

Comment:
We typically say, "No user-facing changes in Logstash core."  
Is that accurate here?